src/openvic-simulation/military/UnitType.cpp

```diff
@@ -322,41 +322,41 @@ bool UnitTypeManager::generate_modifiers(ModifierManager& modifier_manager) cons
 		std::string_view identifier, UnitType::branch_t branch
 	) -> void {
 		const auto stat_modifier = [&modifier_manager, &ret, &identifier](
-			std::string_view suffix, bool is_positive_good, ModifierEffect::format_t format
+			std::string_view suffix, bool is_positive_good, ModifierEffect::format_t format, std::string_view localisation_key
 		) -> void {
 			ret &= modifier_manager.add_modifier_effect(
 				ModifierManager::get_flat_identifier(identifier, suffix), is_positive_good, format,
-				StringUtils::append_string_views("$", identifier, "$: $", suffix, "$")
+				StringUtils::append_string_views("$", identifier, "$: $", localisation_key, "$")
 			);
 		};
 
 		using enum ModifierEffect::format_t;
 
 		ret &= modifier_manager.register_complex_modifier(identifier);
 
-		stat_modifier("attack", true, RAW_DECIMAL);
-		stat_modifier("defence", true, RAW_DECIMAL);
-		stat_modifier("default_organisation", true, RAW_DECIMAL);
-		stat_modifier("maximum_speed", true, RAW_DECIMAL);
-		stat_modifier("build_time", false, INT);
-		stat_modifier("supply_consumption", false, PROPORTION_DECIMAL);
+		stat_modifier("attack", true, RAW_DECIMAL, "ATTACK");
+		stat_modifier("defence", true, RAW_DECIMAL, "DEFENCE");
+		stat_modifier("default_organisation", true, RAW_DECIMAL, "DEFAULT_ORG");
+		stat_modifier("maximum_speed", true, RAW_DECIMAL, "MAXIMUM_SPEED");
+		stat_modifier("build_time", false, INT, "BUILD_TIME");
+		stat_modifier("supply_consumption", false, PROPORTION_DECIMAL, "SUPPLY_CONSUMPTION");
 
 		switch (branch) {
 		case LAND:
-			stat_modifier("reconnaissance", true, RAW_DECIMAL);
-			stat_modifier("discipline", true, PROPORTION_DECIMAL);
-			stat_modifier("support", true, PROPORTION_DECIMAL);
-			stat_modifier("maneuver", true, INT);
-			stat_modifier("siege", true, RAW_DECIMAL);
+			stat_modifier("reconnaissance", true, RAW_DECIMAL, "RECONAISSANCE");
+			stat_modifier("discipline", true, PROPORTION_DECIMAL, "DISCIPLINE");
+			stat_modifier("support", true, PROPORTION_DECIMAL, "SUPPORT");
+			stat_modifier("maneuver", true, INT, "Maneuver");
```

Comment:
```suggestion
			stat_modifier("maneuver", true, INT, "MANUEVER");
```

Reply:
I checked in game, the localisation key is `Maneuver`, it shows that even if you delete every reference to it in the csv files. The non-English versions are in `housedivided2_2.csv` (under the key `Maneuver`).